packages/python/readme_metrics/flask_readme.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime
+import time
+
+from flask import Flask, request
+
+from readme_metrics import MetricsApiConfig
+from readme_metrics.Metrics import Metrics
+from readme_metrics.ResponseInfoWrapper import ResponseInfoWrapper
+
+
+# Guidelines for Flask extensions are available at:
+#     https://flask.palletsprojects.com/en/2.0.x/extensiondev/
+
+
+class ReadMeMetrics:
```

Comment:
Should this class also be `MetricsMiddleware` to match what it is in `django.py`?

Reply:
Again, based on Flask conventions, these are called "extensions" and not "middleware" (and you wouldn't put `Extension` in the class name).